src/main/scala/beam/agentsim/agents/freight/input/PayloadPlansConverter.scala

```diff
@@ -256,7 +388,19 @@ object PayloadPlansConverter {
     currentPlan
   }
 
-  def createPersonId(vehicleId: Id[BeamVehicle]): Id[Person] = Id.createPersonId(s"freight-agent-$vehicleId")
+  def createPersonId(vehicleId: Id[BeamVehicle]): Id[Person] = {
+    if (vehicleId.toString.startsWith(freightIdPrefix)) {
+      Id.createPersonId(s"$vehicleId-agent")
+    } else {
+      Id.createPersonId(s"freight-$vehicleId-agent")
```

Comment:
This and the other create should use the `freightIdPrefix`

Reply:
done